client/src/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App';
+//import App from './components/App';
+import LoginPage from './components/LoginPage'
 import * as serviceWorker from './serviceWorker';
 import configureStore from './store/configureStore';
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './styles/index.css'
 
 const store = configureStore();
+window.store = store;
 
-ReactDOM.render(<App store={store}/>, document.getElementById('root'));
+ReactDOM.render(<LoginPage store={store}/>, document.getElementById('root'));
```

Comment:
you should render App here and use react router to render LoginPage

Reply:
Instead of having App component I implement AppRouter component which will be added with new functionality with CRUD Implementation.

AppRouter history, also it will have Switch cases for different route paths. Now it only  have LoginPage component.